.env.development

```
@@ -8,6 +8,7 @@ LOGOUT_URL=http://localhost:18000/login
 REFRESH_ACCESS_TOKEN_ENDPOINT=http://localhost:18000/login_refresh
 SITE_NAME=edX
 INFO_EMAIL=info@edx.org
+ENVIRONMENT='development'
```

Can we rename it to something that won't conflict with other configs?
```suggestion
ONBOARDING_COMPONENT_ENV='development'
```

---

In prospectus it's `GATSBY_ENVIRONMENT` so it can't be conflict.